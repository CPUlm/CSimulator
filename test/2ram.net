INPUT
OUTPUT o, a
VAR
  a:2, o:2
IN
a = RAM 2 2 01 1 01 11
o = RAM 3 2 01 0 00 00
