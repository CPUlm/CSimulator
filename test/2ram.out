Step 1:
=> o = 00
=> a = 00
Step 2:
=> o = 00
=> a = 11
Step 3:
=> o = 00
=> a = 11
